### DISCUSSION TITLE
Golden Weapons

### DISCUSSION THREAD

**Discussion by Crimson-Haze** [HasangerGames/suroi#430] [Ideas]
I made a golden HP-18. the pump and the stock are made of wood to make it more elegant. 
![Untitled (2)](https://github.com/user-attachments/assets/098213b6-751c-492c-a7fd-38b3818d352f)
let me know what you think.

**Comment by Hashimk2** [comment]
Looks nice
However, is there a change in damage or its stats?
Or is it just for design?
What is the purpose of the Golden HP-18

**Reply by Crimson-Haze** [reply]
its bullet color is now gold and it deals 5% more damage. (I was thinking every weapon has a chance to spawn as golden making it have those stats) if you like this idea I will make more.

**Reply by Hashimk2** [reply]
ask the maintainers if you can add them

**Reply by Crimson-Haze** [reply]
do you think they will do that?

**Reply by Crimson-Haze** [reply]
https://github.com/HasangerGames/suroi/issues/432

I did it

**Comment by Crimson-Haze** [comment]
![Untitled (3)](https://github.com/user-attachments/assets/9dc7cd5f-0d5b-486d-a76b-3a915988c273)
golden uzi

**Comment by Crimson-Haze** [comment]
![Untitled (4)](https://github.com/user-attachments/assets/1778a29f-1094-4356-9e03-da2541d9eea5)
Golden svg

**Reply by Hashimk2** [reply]
What gun is the svg?

**Reply by Crimson-Haze** [reply]
all of them

**Comment by Crimson-Haze** [comment]
Golden Grenade because yes
![Untitled (5)](https://github.com/user-attachments/assets/5c731989-d28f-44bb-80d7-00a23defa4de)
